### ISSUE TITLE
508 Issue from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/pa11y-rails#2]
# Welcome to Pa11y
## Results for https://pages.18f.gov/edu/:
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.3:1. Recommendation: change text colour to #0476d2.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > div > aside > a
  - `<a class="skip-link visuallyhidden focusable" href="#main">Skip to Main Content</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > footer > div > p:nth-child(1) > a
  - `<a href="https://18f.gsa.gov">18F</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > footer > div > p:nth-child(2) > a:nth-child(1)
  - `<a href="https://github.com/18F/pages/">18F Pages</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > footer > div > p:nth-child(2) > a:nth-child(2)
  - `<a href="https://github.com/18F/guides-template/">18F Guides theme</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > footer > div > p:nth-child(2) > a:nth-child(3)
  - `<a href="https://github.com/cfpb/docter/">DOCter</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > footer > div > p:nth-child(2) > a:nth-child(4)
  - `<a href="http://cfpb.github.io/">CFPB</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > footer > div > p:nth-child(3) > a:nth-child(1)
  - `<a href="https://github.com/18F/edu/edit/18f-pages/index.md">Edit this page</a>`
- **Error:** This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 4.36:1. Recommendation: change background to #f5f6f6.
  - WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
  - html > body > div > footer > div > p:nth-child(3) > a:nth-child(2)
  - `<a href="https://github.com/18F/edu/issues">file an issue</a>`
## Summary:
- 8 Errors
- 0 Warnings
- 0 Notices
